l10n_it_codici_carica/README.rst

```diff
@@ -40,6 +40,10 @@ Contributors
 
 * Alessandro Camilli <alessandrocamilli@openforce.it>
 
+Porting from v10:
```

Comment:
Please stick to the [OCA README](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst) format, moreover you can generate a nice readme with [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)